specification/logic/resource-manager/readme.md

```diff
@@ -39,6 +39,16 @@ directive:
 
 ```
 
+### Tag: package-2019-05
+
+These settings apply only when `--package-2019-05` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-05'
+-azure-validator: true
```

Comment:
grammar mistake. You can refer to other RP as reference

Reply:
BTW, you can run linter on your local env by following command `npx autorest --validation --azure-validator --message-format=json specification/logic/resource-manager/readme.md`